dbms/src/Storages/Transaction/ApplySnapshot.cpp

```diff
@@ -473,8 +404,8 @@ void KVStore::handlePreApplySnapshot(const RegionPtrWrap & new_region, TMTContex
     LOG_FMT_INFO(log, "{} apply snapshot success", new_region->toString(false));
 }
 
-template void KVStore::handlePreApplySnapshot<RegionPtrWithBlock>(const RegionPtrWithBlock &, TMTContext &);
```

Comment:
I think we can make these methods to be normal functions instead of template functions after removing the "Block" apply method

https://github.com/pingcap/tiflash/blob/488a64c72eaadc6ece8e266774e5a47389c8ad31/dbms/src/Storages/Transaction/ApplySnapshot.cpp#L407-L412

Reply:
Find we need to keep them for some debug functions